decisions/0009-schema-refactor.md

```diff
@@ -0,0 +1,95 @@
+informed: Emiley Eloe-Fadrosh, Shreyas Cholia, Eric Cavanna 
+---
+# Schema Refactoring: Monterey//Berkeley
+
+## Decisions Made
+
+### Class:PlannedProcess
+* `Class:PlannedProcess` will be created as a root class 
+* All "Activity" classes are now of `Class:PlannedProcess`, and the word "Activity" will be removed
+
+### Class:MaterialProcessing
+* `Class:MaterialProcessing` will be created and be a subclass of `Class:PlannedProcess`
+* All processes involving material to material transformations (Biosamples to ProcessedSample, ProcessedSample to ProcessedSample) will be subclasses of `Class:MaterialProcessing` (examples: `Class:Extraction`, `Pooling`, `MixingProcess`, etc.). 
+* The output of any subclasses of `Class:MaterialProcessing` will have an output of ProcessedSample
+* `Class:ChemicalConversionProcess` will replace `Class:ReactionProcess`
+* `Class:BiosampleProcessing` will be removed. Subclasses of `Class:MaterialProcessing` will replace what was captured. 
+* Linking a protocol to a `Class:MaterialProcess` subclass can happen at the individual class level, or at the aggregation of processes via the `Class:ProtocolExecution`
+* `slot:catalyzed_by` is instantiated on `Class:ChemicalConversionProcess` with `Range:Class:Solution`.
+  * `Class:Solution` has `slot:has_solution_components` with `Range:Class:SolutionComponent` where `slot:compound` has `Range:Class:ProteolyticEnzymeEnum`.
+  * `slot:catalyzed_by` will capture any substrate or enzyme that was used to facilitate a chemical conversion process. 
+  * Enzymes that can be used as catalysts are available in the `Class:ProteolyticEnzymeEnum`.
+  * The `slot:catalyzed_by` will be referenced by proteomics workflows.
+
+### Class:DataGeneration
+* `Class:DataGeneration` will be created and take the place of `Class:OmicsProcessing`
+* `Class:OmicsProcessing` will be removed
+  * All reference to `Class:OmicsProcessing` should be eliminated from NMDC schema & Workflows
+  * Typecode `omprc` will still be valid for `Class:DataGeneration`
+* `Class:DataGeneration` will detail the process of inputting a material sample (Biosample or ProcessedSample, some MaterialEntity) into an instrument and generating data (output)
+  * Instrument data will be denoted using `slot:data_category` on `Class:DataObject`.
+  * The output of DataGeneration should always be `instrument_data`
+* `Class:DataGeneration` will have subclasses `Class:NucleotideSequencing` and `Class:MassSpectrometry` 
+  * This allows for the expansion of additional data types as NMDC grows
+* `slot:instrument_used` will have `Range:Class:Instrument`
+  * `Class:Instrument` will have separate enumerated slots for `vendor` and `model`
+  * `slot:id` will be the identifier of the `instrument_used` and `vendor` and `model` will NOT be inlined for `Class:DataGeneration`. Rather, it will be inferred / traced using the instrument's id.
```

Comment:
From this bullet point alone, it isn't clear to me whether...

`slot:id` will be the identifier of the `instrument_used` and `vendor` ; and `model` will NOT be inlined for `Class:DataGeneration`

or...

`slot:id` will be the identifier of the `instrument_used` ; and `vendor` and `model` will NOT be inlined for `Class:DataGeneration`

I recommend adding a semicolon, for example.

Reply:
`slot:id` will be the identifier of the `instrument_used`; `vendor` and `model` will NOT be inlined for `Class:DataGeneration`. Rather, it will be inferred / traced using the instrument's id.

@eecavanna better?

Reply:
Better - thanks! That eliminated the ambiguity I was referring to. One final change I'd propose is to change "it" to "they" in the new final sentence.

```diff
- Rather, it will be inferred / traced using the instrument's id.
+ Rather, they will be inferred / traced using the instrument's id.
```

...since I think you are talking about multiple things (i.e. `vendor` and `model`).

As far as I'm concerned, the original issue is resolved.

Reply:
resolved in recent commit